libs/Helpers/src/Helpers/logs.cpp

```diff
@@ -1,9 +1,23 @@
+#include "helpers.h"
+
 #include <QDateTime>
 #include <QDebug>
 #include <QString>
 
+#include <iostream>
+
+#include <Helpers/BuildConfiguration.h>
+
+namespace Status::Helpers {
+
 void logFormatter(QtMsgType type, const QMessageLogContext& context, const QString& msg)
 {
+    // TODO: Refactor it into development-tools app
```

Comment:
This can be implemented using [Log4Qt](https://github.com/MEONMedical/Log4Qt)

Reply:
This is a quick work to have some level of debug logs; a follow-up task to handle this in a adequately manner is already define; see [epic #5745](https://github.com/status-im/status-desktop/issues/5745)